.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Pull request details
+
+## List of related issues
+Refs: #ISSUE_NUMBER
+
+
+## Describe the changes made in this pull request
+
+
+## Type of change
+
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Documentation update
```

Comment:
This seems like you are repeating the labels assigned to an issue. Dont think this is needed here as it would make more work

Reply:
I think this can be useful for external contributors.

Reply:
A PR author (external) must already add a link to an issue. They can follow the issue link to get the labels.

Reply:
removed the section

Reply:
thanks